src/domain/tasks/AreaDropdownButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import { Area } from 'types/area'
+
+interface areasNamesAndPks {
+  name: string
+  pk: number
+}
+interface Props {
+  areas: areasNamesAndPks[]
+  addAreaFilter: (areaPk: number) => void
+}
+const AreaDropdownButton = ({ areas, addAreaFilter }: Props): JSX.Element => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  return (
+    <div className="flex justify-center">
+      <div className="flex flex-row ">
+        <button onClick={(): void => setIsOpen(true)} className="block overflow-hidden">
+          <i className="icon-arrow-right text-disabled text-sm md:text-2xl"></i>
+        </button>
+        <div className="underline">Todas las areas</div>
+      </div>
+      {isOpen && (
+        <div className="bg-disabled rounded-lg w-44 mt-4 text-sm md:text-base">
+          <ul>
+            {areas.map((area: areasNamesAndPks) => (
```

Comment:
We need to add a fixed option to select all areas again. Currently once you select an area you cant remove the filter